src/routes/safe/components/Transactions/TxList/TxCollapsed.tsx

```diff
@@ -46,7 +47,7 @@ const TxInfo = ({ info, name }: { info: AssetInfo; name?: string }) => {
       case 'CHANGE_IMPLEMENTATION':
       case 'SET_GUARD':
       case 'DELETE_GUARD':
-        break
+        return <Spacer />
```

Comment:
Have you checked how this affects the transaction list?

Reply:
Yes. This component is only used in the `<TxCollapsed />` and is rendering like before.